docs/openapi.yaml

```diff
@@ -2659,3 +2659,20 @@ paths:
                 $ref: ./entities/tokens/fungible-token.schema.json
               example:
                 $ref: ./entities/tokens/fungible-token.schema.example.json
+
+  /extended/v1/fee_rate:
+    get:
```

Comment:
This should be a POST that requires the serialized tx, so that in the future this endpoint can take advantage of more advanced fee estimation.